### DISCUSSION TITLE
how to implement '/' keyup focus input field

### DISCUSSION THREAD

**Discussion by jitendra1607** [quasarframework/quasar#10275] [General - Components / Directives / etc]
how to implement '/' keyup focus input field like https://v1.quasar.dev

**Comment by jitendra1607** [comment]
`v-on:keyup.191="inpFocus()"`

`inpFocus () {
      this.$refs.searchinput.focus()
    }`

This is what I tried but had no success.

**Reply by pdanpdan** [reply]
Use addEventListener / remove on document
Attach in mounted and remove listener in beforeDestroy
